### PR TITLE
dl/ui from mocked data

### PR DESCRIPTION
**Pull request by darrikonn**
# Overview page from mock data

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by dabbeg — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -6185,12 +6185,12 @@ acorn@^6.4.1:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-6.4.1.tgz#531e58ba3f51b9dacb9a6646ca4debf5b14ca474"
   integrity sha512-ZVA9k326Nwrj3Cj9jlh3wGFutC2ZornPNARZwsNYqQYgN0EsV2d53w5RN/co65Ohn4sUAUtb1rSUAOD6XN9idA==
 
-acorn@^7.0.0:
+acorn@^7.0.0, acorn@^7.1.1:
```
Comment (dabbeg) [inline yarn.lock:6188]: Why the yarn.lock change?

Comment (darrikonn) [reply yarn.lock:6188]: Ran `yarn install` locally which updated the lock file. Most likely the lock file was outdated on `main`, which should've been caught on CI

Comment (darrikonn) [reply yarn.lock:6188]: Gonna do a rebase to verify

Comment (darrikonn) [reply yarn.lock:6188]: Yuup... this is supposed to be valid.

**Review by dabbeg — APPROVED** [review]

**Pull request merged**